test/t/accounts/blocking_test.exs

```diff
@@ -27,9 +29,11 @@ defmodule T.Accounts.BlockingTest do
       assert {:ok, %{match: nil}} = Matches.like_user(reporter.id, reported.id)
       assert {:ok, %{match: %Match{id: match_id}}} = Matches.like_user(reported.id, reporter.id)
 
+      exp_date = expiration_date()
```

Comment:
needed here and everywhere below to record the time wen the push / event occurred (since in push_receive assertion phase the timestamp sometimes is already 1 second up)

even this approach doesn't look ideal, however never managed to break it yet